IonDotnet/Tree/IonSymbol.cs

```diff
@@ -5,6 +5,7 @@ namespace IonDotnet.Tree
     public sealed class IonSymbol : IonText
     {
         private int _sid;
+        private ImportLocation _importLocation;
```

Comment:
If ``_importLocation.Sid`` is the same as ``_sid`` and always present, can we eliminate ``_sid`` and use  ``_importLocation.Sid`` instead?